install/windows/docker-ee.md

```diff
@@ -222,8 +222,8 @@ To update Docker Engine - Enterprise to the most recent release, specify the
 ```powershell
 Install-Package -Name docker -ProviderName DockerMsftProvider -RequiredVersion {{ site.docker_ee_version }} -Update -Force
 ```
-The required version number must match a versions available on the [JSON
-index](https://dockermsft.blob.core.windows.net/dockercontainer/DockerMsftIndex.json)
+The required version number must match a version available on the [JSON
+index](https://download.docker.com/components/engine/windows-server/index.json)
```

Comment:
Thanks, nice catch. 
We should keep the msft link to the JSON file as this is used by the DockerMsftProvider. So, the other link should be updated to be consistent. 

The docker link can also be used to access preview and release candidates which not what we want here in the docs.

Reply:
@StefanScherer No problem, just forced a commit that switched it around, making the msft link consistent in both places. Thanks :)